wagtail/admin/static_src/wagtailadmin/scss/layouts/login.scss

```diff
@@ -114,16 +118,25 @@ input[type=checkbox]:before {
     line-height: 1.6em;
 }
 
+// Forgotten pasword link
 .help {
-    opacity: 1;
-    position: absolute;
     font-size: 1.3em;
-    margin-top: -0.5em;
-    right: 5%;
-    top: 213px;
+    margin-top: 0;
+    padding: 10px 0;
 
     @include media-breakpoint-up(sm) {
-        top: 65%;
+        position: absolute;
+        top: 139px;
```

Comment:
There seems to be a mix of percentages, ems and pixels used for units in this scss file - I've gone with pixels to match the units I was replacing at mobile.